source/frontend/src/features/properties/map/project/add/models.ts

```diff
@@ -0,0 +1,23 @@
+import { Api_Project } from 'models/api/Project';
+import { NumberFieldValue } from 'typings/NumberFieldValue';
+import { toTypeCode } from 'utils/formUtils';
+
+export class ProjectForm {
+  id?: number;
+  projectName?: string;
+  projectNumber?: NumberFieldValue;
+  projectStatusType?: string;
+  region?: NumberFieldValue;
+  summary?: string;
+
+  toApi(): Api_Project {
+    return {
+      id: this.id,
+      code: Number(this.projectNumber) ?? 0,
```

Comment:
this would set the default code to 0. Is that intended?

Reply:
Temporary fix since Database doesn't allow nulls for the code.